app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -0,0 +1,4 @@
+// Note: you have to duplicate $scope in 2 places to make it work
```

Comment:
I just want you to understand WHY you have to duplicate $scope... it's called `dependency injection` ;)